haas/api.py

```diff
@@ -605,20 +702,29 @@ def show_network(network):
 
     network = _must_find(model.Network, network)
 
-    if network.access is not None:
-        auth_backend.require_project_access(network.access)
+    if network.access:
+        #auth tests: admin owned networks with a project access
+        #TODO only network owner allowed to show the network?
```

Comment:
Anyone who can attach to a network needs to be able to run `show_network` in order to get the channels, so this comes down to what other information should be shown.

In terms of other projects with ACLs, I don't have a strong opinion though lean towards the current behavior. For example, if BU has a common provisioning network they let only BU people connect to, it might be ok for them to see one another.

Another aspect is that a user could use the list of other allowed attachers to determine how they treat the network. For example, if I look at the ACL and see only trusted entities, I might not lock down my nodes' interfaces as much as if I saw projects I wasn't familiar with.

For public networks (like the Internet), not showing other projects is reasonable since that would basically let you enumerate the projects.